include/sparrow/array/array_data_factory.hpp

```diff
@@ -490,6 +537,46 @@ namespace sparrow
     }
 
 
+
+    /**
+     * \brief Creates a default array_data object with the specified layout and values and indicate which values are missing.
```

Comment:
```suggestion
     * Creates a default array_data object with the specified layout and values and indicate which values are missing.
```
We use auto-brief :) same suggestion for other comments below